resources/js/store/modules/teams/actions.js

```diff
@@ -27,13 +27,59 @@ export const actions = {
                 });
 
                 context.commit('usersActiveTeam', payload);
+                context.commit('usersTeam', response.data.team);
             }
         })
         .catch(error => {
             console.error('change_active_team', error);
         });
     },
 
+    /**
+     * Leave the team
+     */
+    async LEAVE_TEAM (context, payload)
+    {
+        await axios.post('/teams/leave', {
+            teamId: payload
+        })
+        .then(response => {
+            if (response.data.success) {
+                Vue.$vToastify.success({
+                    title: i18n.t('notifications.success'),
```

Comment:
There is a bug with i18n. During long wait times, the translation doesn't always happen. We need to declare the translations before doing `await`